src/box/sql/func.c

```diff
@@ -522,6 +522,121 @@ func_trim_str(struct sql_context *ctx, int argc, const struct Mem *argv)
 		ctx->is_aborted = true;
 }
 
+static int
+sql_utf8_symbol_count(const char *str, int byte_len)
+{
+	int symbol_count = 0;
+	int str_end = 0;
+	UChar32 c = 0;
+	while (str_end < byte_len) {
+                U8_NEXT((uint8_t *)str, str_end, byte_len, c);
+                symbol_count++;
+        }
+        return symbol_count;
+}
+
+static char*
+sql_kmp(struct sql_context *ctx, const char *str, const int str_size, 
+	                         const char *key, const int key_size,
```

Comment:
Wrong indentation. Please move the arguments to the left so they start right below the very first argument. Also, please remove the trailing space in the line above.